yarn-project/simulator/src/avm/journal/journal.ts

```diff
@@ -66,7 +66,6 @@ type PartialPublicExecutionResult = {
   contractStorageUpdateRequests: ContractStorageUpdateRequest[];
   unencryptedLogsHashes: LogHash[];
   unencryptedLogs: UnencryptedL2Log[];
-  unencryptedLogPreimagesLength: Fr;
```

Comment:
@ AVM team - removed total logs length from journal, as I believe it's not required for you.